packages/block-editor/src/components/button-block-appender/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { Button, Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Inserter from '../inserter';
+
+const ButtonBlockAppender = function( { rootClientId, className = '' } ) {
+	return (
+		<Inserter
+			rootClientId={ rootClientId }
+			renderToggle={ ( { onToggle, disabled, isOpen } ) => (
+				<Button
+					className={ `${ className } button-block-appender` }
```

Comment:
It should be fine as-is, but in aggregating class names, it's conventional to use [`classnames`](https://www.npmjs.com/package/classnames).

```suggestion
					className={ classnames( className, 'button-block-appender' ) }
```

I have a hard time arguing for why you might care one way or the other here, but it did draw my attention as being outside a norm of expectation for how this class is assigned.

Reply:
> I have a hard time arguing for why you might care one way or the other here,

In considering it more: It prevents you from having to assign the default `className = ''`. Often this defaulting is overlooked and results in an assigned class name of something like `"undefined button-block-appender"`, which is usually hard to detect (similar issues #1468 #13192).

Technically it also avoids generating a `className` with leading spaces, as in the default case here you'd produce `" button-block-appender"`. Usually it's not an issue, except in cases when some logic might consider the value as being strictly equal to some expectation.

Reply:
Switched to using classnames 👍